src/ucp/core/ucp_context.c

```diff
@@ -455,6 +455,15 @@ static ucs_config_field_t ucp_context_config_table[] = {
    ucs_offsetof(ucp_context_config_t, reg_nb_mem_types),
    UCS_CONFIG_TYPE_BITMAP(ucs_memory_type_names)},
 
+  {"RMA_OFFLOAD", "try",
```

Comment:
maybe PROTO_OFFLOAD or OFFLOAD_PROTO - to set mode for offload protocols for all cases including HWTM for example?